crates/networking/p2p/peer_handler.rs

```diff
@@ -19,7 +19,7 @@ use crate::{
         },
```

Comment:
To be able to test this better:
- remove the limit here: https://github.com/lambdaclass/ethrex/blob/a00c2920361fb33d0a2c693e4808e877a6b0c974/.github/workflows/pr-main_l1.yaml#L180 
- enable cancun tests: https://github.com/lambdaclass/ethrex/blob/a00c2920361fb33d0a2c693e4808e877a6b0c974/.github/workflows/pr-main_l1.yaml#L166-L171
- Set parallelism to 4 here: https://github.com/lambdaclass/ethrex/blob/a00c2920361fb33d0a2c693e4808e877a6b0c974/.github/workflows/pr-main_l1.yaml#L228

The two last steps can be avided if you wait for https://github.com/lambdaclass/ethrex/pull/4876 and sync with main

Reply:
#4876 has been merged!

Reply:
Merged & removed limit [b82100b](https://github.com/lambdaclass/ethrex/pull/4812/commits/b82100b6199c32b42ef730ead9e9e95a81c00aef)